sdk/python/kfp/dsl/executor.py

```diff
@@ -328,16 +328,24 @@ def execute(self) -> Optional[str]:
             # `Optional[str]`. In this case, we need to strip off the part
             # `Optional[]` to get the actual parameter type.
             v = type_annotations.maybe_strip_optional_from_annotation(v)
-
             if v == task_final_status.PipelineTaskFinalStatus:
                 value = self.get_input_parameter_value(k)
+
+                # TODO: PipelineTaskFinalStatus field names formatted in camel case are deprecated.
```

Comment:
```suggestion
                # PipelineTaskFinalStatus field names pipelineJobResourceName and pipelineTaskName are deprecated. Support for these fields will be removed at a later date.
```

Can you also add this note to the sdk release notes [here](https://github.com/opendatahub-io/data-science-pipelines/blob/master/sdk/RELEASE.md#deprecations)

Reply:
Updated